### PR TITLE
Add Image and PDF uploads

### PR DESCRIPTION
**Pull request by markbreedlove**

### CODE REVIEW

File: app/models/image.rb [hunk 1]
```diff
@@ -1,8 +1,7 @@
 class Image < ActiveRecord::Base
   has_many :attachments, as: :asset
   has_many :sources, through: :attachments
-  validates :file_base, presence: :true
-  validates :mime_type, presence: :true
+  validates :file_name, presence: :true
   validates :height, numericality: { only_integer: true }
   validates :width, numericality: { only_integer: true }
```
Comment (AudreyAltman) [inline app/models/image.rb]: The `:height` and `:width` validations here will cause the upload to fail if their value is `nil`.  We can solve this by adding `allow_blank: true`, so they would read:

`validates :height, numericality: { only_integer: true }, allow_blank: true`
`validates :width, numericality: { only_integer: true }, allow_blank: true`

Comment (markbreedlove) [inline app/models/image.rb]: OK, I'll fix that.  I think I have it setting an empty string, but it's still a good thing to do this.

**Comment by AudreyAltman** [comment]
This looks great.  I successfully an image and a pdf locally.  I only have one small suggestion, and that is to delete two remaining links I found to the now deleted `image:edit` and `document:edit` views.  They are in `app/views/images/show.html.erb` lines 4 and 5, and `app/views/documents/show.html.erb` lines 4 and 5.  

Great job getting this code done in such a short period of time!

**Comment by AudreyAltman** [comment]
Also, it looks like this needs to be rebased to develop.

**Comment by markbreedlove** [comment]
Aha, thanks for catching that.  Making the changes now.  Thanks for your kind words!  I'll rebase it after making the changes.

**Comment by markbreedlove** [comment]
@AudreyAltman: It's rebased and squashed

**Comment by AudreyAltman** [comment]
:dancer:

**Pull request merged**